Lib/abc.py

```diff
@@ -173,6 +173,8 @@ def _dump_registry(cls, file=None):
         for name in sorted(cls.__dict__.keys()):
```

Comment:
sorted() is not needed anymore because dict order is stable.

Other option is whitelist members `('_abc_registry', '_abc_cache', '_abc_negative_cache', '_abc_negative_cache_version')`.

Reply:
I would prefer the whitelist: a suitable order increase also readability...
But I think to stick with `if name.startswith("_abc_")` is easier to maintain.
I was told that this will be rewritten in C, so it's not a big deal.